app/models/setting/email.rb

```diff
@@ -0,0 +1,74 @@
+require 'facter'
+
+class Setting::Email < Setting
+  NON_EMAIL_YAML_SETTINGS = %w(send_welcome_email email_reply_address email_subject_prefix)
+
+  def self.load_defaults
+    Foreman::Deprecation.deprecation_warning('1.16', "email configuration has been moved to settings") if mailconfig.present?
```

Comment:
1. 1.15 at this stage (last stable branch + 2, allows it to be released in two stable releases)
1. Please improve the deprecation warning, specifying that it's email.yaml that has been deprecated.